docs/dbus/DBusAPIReference.lyx

```diff
@@ -597,6 +597,369 @@ Uuid The UUID of the pool.
  Signature: s.
 \end_layout
 
+\begin_layout Description
```

Comment:
Please change heading from ```D-Bus type``` to ```Signature``` and leave out the parenthetical bits in the entries.

Please change "Path to the blockdev" to "Device node".

I'ld prefer it if the states weren't enumerated in the table. That could be left until later (in time and document space, with maybe more of an explanation).

For description of ```BlockDevs``` I'ld prefer
```Summary of the characteristics and current state of each of the block devices that make up the pool. Each block device is described by the following attributes:```

You might want to add an additional column for mandatory vs. optional in the table and leave out the sentence at the end.